packages/typespec-client-generator-core/src/example.ts

```diff
@@ -197,7 +194,7 @@ function handleHttpOperationExamples(
       kind: "http",
       name: title,
       description: title,
-      filePath: example.file.path,
+      filePath: example.relativePath,
```

Comment:
where do those examples come from?

Reply:
from config of `examples-directory`, this `filePath` is a relative path to this config for each of the example.

Reply:
Does this config has the format: absolute-path so the compiler validate that for you. Relative path are a nightmare to understand to what and we agreed a while back to not have any new thing using it

Reply:
how to define the option defined in the tcgc which is not an emitter?

Reply:
where is it setup then? its not the emitter sending it?
For doc on how to use it https://typespec.io/docs/next/extending-typespec/emitters#absolute-path

Reply:
talked to Izzy and we looked at the code, the confusion comes from teh fact that you hardcode that directly. WHere then it makes a lot more sense to hard code to `{project-root}/examples`. However that directory is configurable by the autorest emitter so it is not necessarly always in this folder FYI.

Reply:
i don't quite understand what you mean. `examples-directories` will be configured in each language's emitter. and the `filePath` here is to show the relative path of one example to that config path. so, join `examples-directories` and `filePath` will get the full absolute path of the example.

Reply:
this part `examples-directories will be configured in each language's emitter.`. There is 2 things about it
1. Need to make sure you use `format: absolute-path` here so this issue doesn't keep happening
2. This is a lot of duplication, every emitter will need to respecify this property in the tspconfig.yaml
   - not much we can do abotu that right now

Reply:
for 1, since it is a config in emitter, i'll let all languages to use the right format, anything could be done in tcgc?

Reply:
Yeah no just more of a FYI to make sure everyone does the right thing

Reply:
i found in current [autorest emitter](https://github.com/Azure/typespec-azure/blob/5edf99de54d93a4ad04303ae5dca3d507081d557/packages/typespec-autorest/src/lib.ts#L152), the `examples-directories` not set format to be `absolute-path`. can we change that? if so, i believe we will enforce all pr merged to azure spec repo to set it with absolute path.

Reply:
yeah we have issue to do that, just it is a breaking change as this existed before `absolute-path` was present
the goal is to rename it to `examples-dir`

Reply:
which would be nice if you named it that way as well as we have been naming things with `-dir` for directories

Reply:
got it, let me do the change.